datafusion/sql/Cargo.toml

```diff
@@ -39,6 +39,7 @@ unicode_expressions = []
 [dependencies]
 arrow = { workspace = true }
 arrow-schema = { workspace = true }
+chrono = { workspace = true }
```

Comment:
I know we are sensitive to new dependencies so wanted to flag this proposed addition to datafusion-sql @alamb ...

It is used in this PR to facilitate converting arrow Date types to a formatted string which can be placed in a SQL expression like `CAST('2024-01-01' to DATE)`

Reply:
It may be a good solution to put the unparser behind a feature flag and then any unparser specific dependencies can only be enabled if one needs unparser support.

Reply:
I think arrow already depends on chrono so this change doesn't add any new actual dependency (just an explicit one)

Reply:
I went ahead and put unparser behind a (default) feature flag anyway. Could come in handy if we need a more invasive dependency later or someone just does not want this functionality.